web-app/src/pages/Login.vue

```diff
@@ -61,6 +61,8 @@ export default class Login extends Vue {
   private async mounted() {
     if (this.$route.query.code) {
       await api.auth.getCookieToken(this.$route.query.code as string);
+
+      await this.$router.push({ name: 'TournamentsList' });
```

Comment:
```suggestion
      await this.$router.push({ name: 'TournamentList' });
```

I'm not sure, that is best way to change routes. 
You definitely shouldn't use `push` only `replace`. It's silly to allow user to back to login page